configure.ac

```diff
@@ -18,9 +18,9 @@ AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [Gettext package])
 AM_GLIB_GNU_GETTEXT
 IT_PROG_INTLTOOL([0.40.0])
 
-AM_PATH_PYTHON_VERSION(2.7, 2.7.0)
+AM_PATH_PYTHON_VERSION(3.6, 3.6.0, 3.5, 3.5.0, 2.7, 2.7.0)
```

Comment:
Version 2.7 shouldn't be valid anymore after porting to Python 3.

I have a feeling we should just drop the whole `acinclude.m4` which contains `AM_PATH_PYTHON_VERSION`, and just use `AM_PATH_PYTHON`.